ui/packages/catalog-ui-search/src/main/webapp/js/controllers/cesium.layerCollection.controller.js

```diff
@@ -100,37 +108,50 @@ var Controller = CommonLayerController.extend({
     }
   },
   setAlpha: function(model) {
-    var layer = this.layerForCid[model.id]
+    const layer = this.layerForCid[model.id]
     layer.alpha = model.get('alpha')
   },
   setShow: function(model) {
-    var layer = this.layerForCid[model.id]
+    if (!this.layerForCid[model.id]) {
+      this.initLayer(model)
```

Comment:
So now layers aren't initialized unless we show them?  And the first show is when they are initialized?

Reply:
Yes, apparently that is the expected behavior.